hail/python/hailtop/fs/fs_utils.py

```diff
@@ -31,6 +38,25 @@ def open(path: str, mode: str = 'r', buffer_size: int = 8192) -> io.IOBase:
     ...     for line in f:
     ...         print(line.strip())
 
+    Access a text file stored in a Requester Pays Bucket in Google Cloud Storage:
+
+    >>> with hfs.open( # doctest: +SKIP
+    ...     'gs://my-bucket/notes.txt',
+    ...     requester_pays_config='my-project'
+    ... ) as f:
+    ...     for line in f:
+    ...         print(line.strip())
+
+    Specify multiple Requester Pays Buckets within a project that are acceptable
+    to access:
+
+    >>> with hfs.open( # doctest: +SKIP
+    ...     'gs://my-bucket/notes.txt',
+    ...     requester_pays_config=('my-project', ['my-bucket', 'bucket-2'])
+    ... ) as f:
+    ...     for line in f:
+    ...         print(line.strip())
```

Comment:
I feel like this should only support `requester_pays_project`, since you can't `open` from more than one bucket.

Reply:
AFAICT the only method where it would be possible to specify multiple buckets would be `copy`. It's perhaps worth asking though if anyone would ever want to specify two different requester pays projects when copying between two buckets. Seems kind of weird and like it's leading to some more annoying bookkeeping than if we just had a `Dict[str, FS]`.

Reply:
good point; checked in with @danking and he said it's not inconceivable that someone would want to put together one big requester pays config and just pass the same object into multiple invocations of `open`, so hopefully this latest change reduces the bookkeeping enough that it's not too big of an issue to support the `Tuple[str, List[str]]` case of the type